packages/frontend/src/tests/test-utils.tsx

```diff
@@ -1,55 +1,7 @@
 // currently working on wrapper (if needed for future tests)
-import React, { FC, ReactElement } from 'react'
```

Comment:
Do we still need this file? It doesn't seem to be imported anywhere.

Reply:
Will delete that considering I am not using that now. Was going to use it to create an abstraction to keep code DRY but may not need it. And will merge the two directories in a commit with `testPathIgnorePatterns` option in jest config.